README.md

```diff
@@ -63,16 +65,15 @@ Usage:  <br>
 | ----------- | ----------- |
 |`--ldc`    | validate against the LDC ontology |
 |`--program`| validate against the program ontology |
-|`--ont FILE ...` | validate against the OWL-formatted ontolog(ies) at the specified filename(s) |
+|`--ont=FILE ...` | validate against the OWL-formatted ontolog(ies) at the specified filename(s) |
 |`--nist` | validate against the NIST restrictions |
 |`--nist-ta3` | validate against the NIST hypothesis restrictions (implies `--nist`) |
-|`--abort [num]` | Abort validation after `[num]` SHACL violations (num > 2), or three violations if `[num]` is omitted. |
+|`--abort=[num]` | Abort validation after `[num]` SHACL violations (num > 2), or three violations if `[num]` is omitted. |
```

Comment:
It's `--abort[=num]` like in the Usage.

Reply:
Copying and pasting is fun!

Reply:
fixed

Reply:
Oh my, how did I miss that?  I was looking for it in `--pm` in the README for some reason.  My bad.